### ISSUE TITLE
Configuration for rule "no-implicit-coercion" is invalid

### ISSUE THREAD

**Issue by rickhedin** [AtomLinter/linter-eslint#518]
I get this message in Atom: 

```
Error: /Users/rhedin/new_world/asa/node_modules/webapps-tools-code-standards/.eslintrc: Configuration for rule "no-implicit-coercion" is invalid: Value "data["1"].allow" has additional properties.

Referenced from: /Users/rhedin/new_world/asa/.eslintrc
```
But the rule for no-implicit-coercion looks correct: 

```
    "no-implicit-coercion": [1, {
      "boolean": true,
      "number": true,
      "string": true,
      "allow": ["!!"]
    }],                                 // disallow the type conversions with shorter notations
```
linter-eslist  7.1.3
atom  1.6.1 

Thoughts?

**Comment by Arcanemagus** [comment]
What version of `eslint` are you using? How is it installed?

**Comment by rickhedin** [comment]
Hmm.  I hope this is being helpful.

```
rhedin@RHEDIN-MBP:asa (asa147_selector_bug=)$ eslint --version
v2.4.0
rhedin@RHEDIN-MBP:asa (asa147_selector_bug=)$ which eslint
/Users/rhedin/.nvm/versions/node/v5.6.0/bin/eslint
rhedin@RHEDIN-MBP:asa (asa147_selector_bug=)$ node --version
v5.6.0
rhedin@RHEDIN-MBP:asa (asa147_selector_bug=)$ 
```

**Comment by Arcanemagus** [comment]
Do you have the "Use global eslint" option enabled?

**Comment by rickhedin** [comment]
Hi.  No, I don't.  But there's a larger story. 

I couldn't find the "Use global ESList installation" checkbox.  All I saw was the readme.  So I followed the hints here:  https://github.com/AtomLinter/linter-eslint/issues/461

When I uninstalled and reinstalled linter-eslint and re-started Atom, not only did the option appear in the settings, but the spurious complaint about the no-implicit-coercion rule went away. 

I need to be quicker to uninstall and reinstall, I guess. 
Thanks for your help, Arcanemagus.         --  Rick
